src/main/java/network/brightspots/rcv/CsvCvrReader.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * RCTab
+ * Copyright (c) 2017-2022 Bright Spots Developers.
```

Comment:
Realized that we'll need to update the copyright dates to 2023. There's an auto setting for this when committing files in IntelliJ "Update copyright", but maybe you're not using IntelliJ.

Maybe create a new required issue for this before we launch 1.4.0?

Reply:
Should we update all files, or only files as they're edited? I can find that intellij setting -- I'm using vim currently but I do have access to intellij.

Reply:
Created #668 for this.